packages/tspd/test/emitters/markdown/model.test.ts

```diff
@@ -0,0 +1,205 @@
+import { ok } from "assert";
+import { describe, expect, it } from "vitest";
+import { renderMarkdowDoc } from "../../../src/ref-doc/utils/markdown.js";
+import { createMarkdownRenderer } from "../../test-utils.js";
+
+async function renderModel(code: string) {
+  const { renderer, refDoc } = await createMarkdownRenderer(`namespace Lib; ${code}`);
+  const fooModel = refDoc.namespaces[0].models.find((x) => x.name === "Test");
+  ok(fooModel, "Expected a model named `Test`");
+  return renderMarkdowDoc(renderer.model(fooModel));
+}
+
+it("render simple model", async () => {
+  const result = await renderModel(`model Test {}`);
+  expect(result).toEqual(
+    [
+      "# `Test`",
+      "",
+      "",
+      "```typespec",
+      "model Lib.Test",
+      "```",
+      "",
+      "",
+      "## Properties",
+      "| Name | Type | Description |",
+      "|------|------|-------------|",
```

Comment:
Is this really what we would want a model with no properties to emit though? I would think we'd either want it to not emit the properties section or (my preference) just emit a concise statement that this model has no properties.